src/test/java/org/cactoos/list/ImmutableTest.java

```diff
@@ -560,4 +560,30 @@ public void testToString() {
             new IsEqual<>(new ListOf<>("a", "b", "c").toString())
         ).affirm();
     }
+
+    @Test
+    public void subListReturnsListIteratorWithSupportedSet() {
+        new Assertion<>(
+            "subList.listIterator().set() must throw exception",
```

Comment:
@andreoss `subList` from who? Maybe you can be more explicit here.

Reply:
@fabriciofx Done.